projects/storefrontlib/src/cms-components/order-confirmation/components/order-confirmation-thank-you-message/order-confirmation-thank-you-message.component.ts

```diff
@@ -15,7 +15,9 @@ import { tap } from 'rxjs/operators';
 })
 export class OrderConfirmationThankYouMessageComponent
   implements OnInit, OnDestroy {
-  order$: Observable<Order>;
+  order$: Observable<any>;
```

Comment:
Because of union types. If I would of put Order | ReplenishmentOrder, it would cause a type error in the template since it's inclusive and doesn't take all properties when doing above.